app/views/idv/shared/_document_capture.html.erb

```diff
@@ -7,7 +7,7 @@
 <%= tag.div id: 'document-capture-form', data: {
       app_name: APP_NAME,
       liveness_required: nil,
-      mock_client: (DocAuthRouter.doc_auth_vendor(discriminator: document_capture_session_uuid) == 'mock').presence,
+      mock_client: ['mock', 'true_id_mock'].include?(DocAuthRouter.doc_auth_vendor(discriminator: document_capture_session_uuid)).presence,
```

Comment:
what if we added a `mock?` method to the interface of the proofer classes?

Reply:
@zachmargolis ,thanks for the feedback, this is an exploratory effort for now.